CodeReview/src/commands/CCodeReviewComment.cpp

```diff
@@ -49,12 +54,24 @@ bool CCodeReviewComment::canInterpret(Visualization::Item*, Visualization::Item*
 	return false;
 }
 
-Interaction::CommandResult* CCodeReviewComment::execute(Visualization::Item*, Visualization::Item*,
-				const QStringList&, const std::unique_ptr<Visualization::Cursor>& cursor)
+Interaction::CommandResult* CCodeReviewComment::execute(Visualization::Item* source, Visualization::Item*,
+				const QStringList&, const std::unique_ptr<Visualization::Cursor>&)
 {
-	auto cursorOwner = cursor->owner();
-	auto overlay = new CodeReviewCommentOverlay{cursorOwner};
-	cursorOwner->addOverlay(overlay, "CodeReviewComment");
+	for (auto manager : Model::AllTreeManagers::instance().loadedManagers())
+		if (manager->nodeIdMap().contains(source->node()))
+		{
+			auto id = manager->nodeIdMap().id(source->node()).toString();
+			auto commentedNode = CodeReviewManager::instance().commentedNode(id);
+			commentedNode->commentNodes()->append(new Comments::CommentNode{"comment here"});
+
+			// only create highlight if not already existent
```

Comment:
If a comment highlight already exists, should we really be creating any new comment nodes? Shouldn't this be a noop in that case?